src/main/java/frc/robot/subsystems/Vision/VisionIOReal.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Pose3d;
+import org.photonvision.PhotonCamera;
+
+/** Add your docs here. */
+public class VisionIOReal implements VisionIO {
+  // constants
+  public static final String CAMERA_NAME = "opi-camera";
```

Comment:
this shouldnt be hardcoded, pass it through the constructor

Reply:
which constructor?